src/displayapp/apps/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ else ()
     set(DEFAULT_USER_APP_TYPES "${DEFAULT_USER_APP_TYPES}, Apps::Paint")
     set(DEFAULT_USER_APP_TYPES "${DEFAULT_USER_APP_TYPES}, Apps::Paddle")
     set(DEFAULT_USER_APP_TYPES "${DEFAULT_USER_APP_TYPES}, Apps::Twos")
+    set(DEFAULT_USER_APP_TYPES "${DEFAULT_USER_APP_TYPES}, Apps::Dice")
```

Comment:
I don't think we want this to be a default app when merging, but we can leave it enabled for now for seeing the memory and flash usage from the tests.

Reply:
Sounds good! :slightly_smiling_face: I just included it because it seemed like most of the user apps were default currently (and it makes it 12 apps to fill up both screens of the list). But I think both options make sense, am happy with what the maintainers feel is most appropriate there!